src/extlibs/er_force_sim/src/amun/simulator/simulator.h

```diff
@@ -106,14 +134,28 @@ class camun::simulator::Simulator : public QObject
      * @return simulator state
      */
     world::SimulatorState getSimulatorState();
-    void handleSimulatorSetupCommand(const std::shared_ptr<amun::Command> &command);
+
+    /**
+     * Handles a simulator set up command and configure the simulator accordingly
+     *
+     * @param command the simulator set up command
+     */
+    void handleSimulatorSetupCommand(const std::unique_ptr<amun::Command> &command);
```

Comment:
No issues, but I was just wondering why the change here was made to swap from shared_ptr to unique_ptr?  Is there an advantage to using a unique_ptr here instead of a shared_ptr?

Reply:
yeah unique_ptrs limit where the data can be accessed. It's a code quality thing that we'll have a presentation on at some point